modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc

```diff
@@ -8,38 +8,40 @@
 ifdef::context[:parent-context: {context}]
 
 // Base the file name and the ID on the assembly title. For example:
-// * file name: my-assembly-a.adoc
-// * ID: [id="my-assembly-a"]
-// * Title: = My assembly A
-
-// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
-[id="a-collection-of-modules"]
-// If the assembly is reused in other assemblies in a guide, include {context} in the ID: [id="a-collection-of-modules_{context}"].
-= A collection of modules
+// * file name: assembly-my-user-story.adoc
+// * ID: [id="assembly-my-user-story_{context}"]
+// * Title: = My user story
+
+// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken. Include {context} in the ID so the assembly can be reused:
+[id="assembly-my-user-story_{context}"]
+= My user story
 //If the assembly covers a task, start the title with a verb in the gerund form, such as Creating or Configuring.
 :context: assembly-keyword
 // The `context` attribute enables module reuse. Every module's ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
 
-This paragraph is the assembly introduction. It explains what the user will accomplish by working through the modules in the assembly and sets the context for the user story the assembly is based on. Can include more than one paragraph. Consider using the information from the user story.
+[role="_abstract"]
+This paragraph is the assembly introduction. It explains what the user will accomplish by working through the modules in the assembly and sets the context for the user story the assembly is based on. The `[role="_abstract"]` tag defines the first paragraph of the introduction for search metadata.
 
 .Prerequisites
+//Prerequisites are optional. Delete if your assembly has no prerequisites.
 
-* A bulleted list of conditions that must be satisfied before the user starts following this assembly.
-* You can also link to other modules or assemblies the user must follow before starting this assembly.
-* Delete the section title and bullets if the assembly has no prerequisites.
+* X is installed. For information about installing X, see <link>.
```

Comment:
"X is installed. For information about installing X, see <link>" seems wordy and redundant. Use link instead. 
```suggestion
* link:<url>[X is installed.] 
```

Reply:
The proposed format is also more consistent with .Additional resources

Reply:
OK, Well, I just thought I was giving a practical example. The focus of this PR is to get the attributes in so I've reinstated the original prerequisite guidance and we can look at this later if we want.

Reply:
@rolfedh @emmurphy1 I like brevity, but the "For information about" is important to provide context. It gives me a sense of security about what I'm clicking on. "X is installed" does not necessarily inform me that by clicking on it, I will go to a page that describes how to install X. Generally, a link to a document should include the title of the document. See [Link text](https://developers.google.com/style/link-text) in the Google Developer Documentation Style Guide.

Reply:
@stoobie So you support saying "X is installed. For information about installing X, see [Installing X](https://developers.google.com/style/link-text)" example?

Reply:
My preferred way would be to drop "about installing X" because that's already implied by "Installing X", the label of the link. The result:

"X is installed. For information, see <Installing X>."

Reply:
@rolfedh I agree with @mrksu. If it's under Additonal resources, I'm good with just the title of the target.

Reply:
@stoobie @mrksu @rolfedh all good discussion but I've removed this update from this PR for expediency. The new issue is here:
https://github.com/redhat-documentation/modular-docs/issues/135

Reply:
I have copied the content of this conversation to the new issue so resolving this conversation.